content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -60,7 +50,7 @@ to manage the lifecycle of Istio.
 1. If a service account has not already been installed for Tiller, install one:
 
     {{< text bash >}}
-    $ kubectl apply -f install/kubernetes/helm/helm-service-account.yaml
+    $ kubectl apply -f @install/kubernetes/helm/helm-service-account.yaml@
```

Comment:
what is the purpose of @ here?

Reply:
Put @ on either end of the yaml file path, this will generate a link to the file on GitHub.

Reply:
yes I know, but why link it?